tla-io/src/main/scala/at/forsyte/apalache/io/annotations/parser/AnnotationLexer.scala

```diff
@@ -61,37 +57,21 @@ object AnnotationLexer extends RegexParsers {
 
   private def string: Parser[STRING] = {
     """"[a-zA-Z0-9_~!@#\\$%^&*\-+=|(){}\[\],:;`'<>.?/ \t\r\f\n]*"""".r ^^ { name =>
-      STRING(removeLeadingCommentsAndControlChars(name.substring(1, name.length - 1)))
+      STRING(name.substring(1, name.length - 1))
     }
   }
 
   private def inline_string: Parser[INLINE_STRING] = {
-    """:[a-zA-Z0-9_~!#\\$%^&*\-+=|(){}\[\],:`'<>.?/ \t\r\f\n]*;""".r ^^ { name =>
-      INLINE_STRING(removeLeadingCommentsAndControlChars(name.substring(1, name.length - 1)))
+    """:[a-zA-Z0-9_~!#\\$%^&*\-+=|(){}\[\],:`'<>.?/ \t\r\f\n]*;""".r ^^ { text =>
```

Comment:
This looks like the same regex as on line 59. Can we instead share a value?

Reply:
They look very similar, but they are different. The latter one does not admit `@` and `;`. I will add a comment.

Reply:
Ah. Please add a comment noting the difference? Otherwise it'll be up to a human to carefully parse the regex and they are liable to make the same mistake I did.